CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ CHANGELOG
 Swift 5.1
 ---------
 
+* [SR-1340](https://bugs.swift.org/browse/SR-1340):
+
+`Self` can now be used inside member functions and for function arguments of structs and enums to refer to the containing type.
```

Comment:
Indent by two spaces, to become another paragraph of the list item.

Reply:
```suggestion
  `Self` can now be used inside member functions and for function arguments of structs and enums to refer to the containing type.
```

Reply:
Thanks Ben